kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -128,110 +155,107 @@
-        <antcall target="marshallers-config" />
+        <antcall target="marshallers-plugins" />
 
         <!-- Wires bundles config -->
-        <antcall target="wires-config" />
+        <antcall target="wires-plugins" />
         <available
             file="${project.build.directory}/plugins/org.eclipse.kura.wire.component.conditional.provider_${org.eclipse.kura.wire.component.conditional.provider.version}.jar"
             property="is.nashorn.conditional.component.present" />
         <antcall target="wires-config-nashorn-js-engine" />
 
         <!-- Miscellaneous bundles config -->
-        <antcall target="misc-config" />
+        <antcall target="misc-plugins" />
 
         <!-- Keystore Management bundles config -->
-        <antcall target="keystore-management-config" />
+        <antcall target="keystore-management-plugins" />
 
         <!-- Filesystem log provider bundle config -->
-        <antcall target="filesystem-logprovider-config" />
+        <antcall target="filesystem-logprovider-plugins" />
         
-        <antcall target="container-orchestrator-config" />
+        <antcall target="container-orchestrator-plugins" />
 
-        <antcall target="configuration.change.manager-config" />
+        <antcall target="configuration.change.manager-plugins" />
 
-        <antcall target="event-publisher-config" />
+        <antcall target="event-publisher-plugins" />
 
         <!-- Conditional entries. Check for jar presence, and invoke relevant ant
             target -->
         <available
             file="${project.build.directory}/plugins/org.eclipse.kura.web2_${org.eclipse.kura.web2.version}.jar"
             property="is.web2.jar.available" />
-        <antcall target="web2-config" />
+        <antcall target="web2-plugins" />
 
         <available
             file="${project.build.directory}/plugins/org.eclipse.kura.linux.gpio_${org.eclipse.kura.linux.gpio.version}.jar"
             property="linux.gpio.jar.present" />
-        <antcall target="gpio-config" />
-        <antcall target="emulator-gpio-config" />
-        <antcall target="emulator-config" />
-        <antcall target="linux-network-config" />
-
-        <!-- Add mToolkit to config.ini -->
-        <propertyfile
-            file="${project.build.directory}/${build.output.name}/config.ini">
-            <entry key="osgi.bundles" operation="+"
-                value=", reference:file:${kura.install.dir}/${kura.symlink}/${plugins.folder}/org.tigris.mtoolkit.iagent.rpc_3.0.0.20110411-0918.jar@5:start" />
-        </propertyfile>
+        <antcall target="gpio-plugins" />
+        <antcall target="emulator-gpio-plugins" />
+        <antcall target="emulator-plugins" />
+        <antcall target="linux-network-plugins" />
+
+        <!-- Add mToolkit to distribution -->
+        <antcall target="install-plugin">
+            <param name="bundle.file" value="org.tigris.mtoolkit.iagent.rpc_3.0.0.20110411-0918.jar" />
```

Comment:
🤔 I don't have the complete picture so this might be a dumb question: is it absolutely necessary to have this hardcoded?

Reply:
Classic! 😄 

We can keep this as-is (unless you have the time/will to investigate this further... but for me the PR is OK)